src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -889,6 +901,32 @@ class PublishArgument(cli.Argument):
             'required': False,
         },
     )
+    EXPIRE_BUILDS = cli.ArgumentProperties(
```

Comment:
This argument is added to defined for action `app-store-connect builds submit-to-testflight`, but not defined in method signature. As a result, when invoking
```shell
app_store_connect builds submit-to-testflight $BUILD_ID --expire-builds
```

following exception occurs:

```python
[14:46:32 01-02-2023] ERROR cli_app.py:116 > Exception traceback:
Traceback (most recent call last):
  File "/Users/priit/nevercode/cli-tools/src/codemagic/cli/cli_app.py", line 204, in invoke_cli
    CliApp._running_app._invoke_action(args)
  File "/Users/priit/nevercode/cli-tools/src/codemagic/cli/cli_app.py", line 160, in _invoke_action
    return cli_action(**action_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/priit/nevercode/cli-tools/src/codemagic/cli/cli_app.py", line 428, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
TypeError: BuildsActionGroup.submit_to_testflight() got an unexpected keyword argument 'expire_builds'
```

Reply:
This argument is not used any more and can be removed.